tests/gql_behave/tests/memgraph_V1/features/aggregations.feature

```diff
@@ -237,6 +277,16 @@ Feature: Aggregations
             | 3 | 0   |
             | 4 | 1   |
 
+    Scenario: Max test 04:
+        Given an empty graph
+        When executing query:
+        """
+        RETURN max(null)
+        """
+        Then the result should be:
+            | max(null) |
+            | null      |
+
```

Comment:
I like the new test cases! First I thought I can construct a case which breaks the compatibility with openCypher/Neo4j:

```
memgraph> MATCH (n) DELETE n;
Empty set (0.001 sec)
1 nodes have been deleted.
memgraph> CREATE (n {property: 1});
Empty set (0.001 sec)
1 nodes have been created.
memgraph> CREATE (n);
Empty set (0.001 sec)
1 nodes have been created.
memgraph> MATCH (n) RETURN count(n) < n.property;
+-----------------------+
| count(n) < n.property |
+-----------------------+
| Null                  |
| false                 |
+-----------------------+
2 rows in set (0.001 sec)
```
But, fortunately I was shocked that is the exact result Neo4j returns. So this test is also passed. However I would introduce one more test that checks the case when the `group_by` expression sometimes gets evaluated to `Null`, something with the following queries:
```
CREATE (n);
CREATE (n {property: 1});
MATCH (n) RETURN count(n) < n.property, count(n.property), count(n), avg(n.property), min(n.property), max(n.property), sum(n.property);
```
And the expected result should be:
```
+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+
| count(n) < n.property | count(n.property)     | count(n)              | avg(n.property)       | min(n.property)       | max(n.property)       | sum(n.property)       |
+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+
| Null                  | 0                     | 1                     | Null                  | Null                  | Null                  | 0                     |
| false                 | 1                     | 1                     | 1                     | 1                     | 1                     | 1                     |
+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+
```

Reply:
I am thinking if the ordering of the rows might cause some problems, because for me its the other way around, so:
+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+
                               │| count(n) < n.property | count(n.property)     | count(n)              | avg(n.property)       | min(n.property)       | max(n.property)       | sum(n.property)       |
                               │+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+
                               │| false                 | 1                     | 1                     | 1                     | 1                     | 1                     | 1                     |
                               │| Null                  | 0                     | 1                     | Null                  | Null                  | Null                  | 0                     |
                               │+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+

Reply:
I copied it from Memgraph, but I might created the vertices in reversed order.